### DISCUSSION TITLE
CS236: Deep Generative Models

### DISCUSSION THREAD

**Discussion by jjinCode** [Pseudo-Lab/9th-builder#11] [plan]
### 프로젝트 목표
1. 오픈강의를 통한 Autoregressive, GAN, Diffusion 등 다양한 생성모델에 대한 이론적 이해
2. 생성모델 관련 프로그래밍 과제(course assignments) or 프로젝트(코드리뷰/논문리뷰) 진행

### 프로젝트 내용
![Untitled (10)](https://img1.daumcdn.net/thumb/R1280x0/?scode=mtistory2&fname=https%3A%2F%2Fblog.kakaocdn.net%2Fdn%2Fm3cL0%2FbtqXYo4PaoQ%2FNuXl4DeyRw7tXycZS1ZXy1%2Fimg.png)
- Standford Online에 올라온 강의를 기반으로 진행되는 생성모델 이론 스터디 (강의 영상 + 노트)
- 코드레벨에서의 생성모델의 이해

## 모집 대상

**지원 대상**
- 생성모델 학습에 관심이 있고 업무/과제에 활용하길 원하시는 분
- 기본적인 Machine Learning, Deep Learning에 대한 이해가 있으신 분
- 학부 수준의 수학 지식을 갖추신 분 (스터디에 활용하려는 강의가 박사과정 코스웍 수준)
- 기본적인 Python, Pytorch를 활용한 프로그래밍 경험이 있으신 분
- **✨스터디에서 공부하는 내용을 잘 정리해나갈 의지가 있으시고 스터디에 참여 할 여유시간이 되시는 분✨**

**모집 인원**
- 5~6명

### 첫 모임 날짜 & 시간 & 장소
- 날짜: 9월 10일
- 시간: 오후 9시
- 장소: 온라인 [#ROOM-YB](https://discord.com/channels/944032730050621450/1068785185686032414)

### 정기 모임 시간 & 장소

- 시간: 매주 화요일 오후 9시
- 장소: 온라인 [#ROOM-YB](https://discord.com/channels/944032730050621450/1068785185686032414)

### 학습 자료
![image](https://github.com/user-attachments/assets/9ee1d388-8dcb-4c76-8f78-4687dc2d6f5e)
[Stanford CS236: Deep Generative Models I 2023](https://www.youtube.com/playlist?list=PLoROMvodv4rPOWA-omMM6STXaWW4FvJT8)
- Stanford에서 공개한 23년도 Deep Generative Model 강의와 학습자료입니다.
- 제공되는 강의 영상과 자료들을 바탕으로 매주 학습을 진행합니다.
- 매주 돌아오는 스터디날에 정해진 발표순서 대로 돌아가면서 (발표자가 강사가 된 것 처럼) 내용 설명을 진행합니다.
- 발표자를 제외한 나머지 인원은 질문을 하거나 발표자의 내용에 대한 토론을 진행합니다.

### 스터디 활동 방법
💡 **오픈 아카데미** 형태(=청강 가능)로 진행하며, 발표자료는 가짜연구소를 통해 **공개적으로 배포할** 수 있도록 할 예정입니
    
1. **진행 & 발표**
    - 출석체크를 위해 **실명**으로 참여 부탁드립니다.
    - 온라인에서 여러분을 만나는 환경이므로 **웹캠**을 꼭 준비해 주세요~
    - 발표자는 OT 날 선정 (참여 인원에 따라 1인 1-2회 정도 발표 예상)
    
2. **학습 활동 (필수)**
    - 매주 학습 분량 만큼 발표자가 발표 진행  
    - 주간 학습한 내용/자료를 스터디 프로젝트 깃헙에 정리
    -  **한 주에 한 강 씩 진도 나가기에 급급하지 않은 생성모델에 대한 깊은 이해와 토론 목표**
    - **학습 내용에 대한 질문 및 공유**

## 주차별 목표와 학습 내용
💡 주차별 진행계획은 러너(스터디원)분들과 협의에 의해 변경될 수 있습니다.

| 날짜 | 내용 | 발표자 | 매체 | 
| ----- | ----- | -------- | ----- |
| 2024/09/10 | OT (스터디 소개, 방향 논의, Ice breaking) | [박진호]() | [링크]() | 
| 2024/09/17 | Lecture 1 - Introduction | [박진호]() | [링크]() | 
| 2024/09/24 | Lecture 2 - Background | [박진호]() | [링크]() |

**Comment by thrcle** [comment]
심도있는 스터디가 될 것 같네요 화이팅입니다!

**Comment by WongiPark0628** [comment]
기회가 된다면 청강할게요..!
